### ISSUE TITLE
Fix TOUGH2 output

### ISSUE THREAD

**Issue by daniellivingston** [lanl/voronoi#3]
On 3D tetrahedral meshes, it appears that output is corrupted.

```
ELEME----1----*----2----*----3----*----4----*----5----*----6----*----7----*----8
AAA01          ^@P<C0>'^A0.1822E-040.1822E-04          0.0000E+000.0000E+000.0000E+00
```

Critical bug that needs fixing.

**Comment by daniellivingston** [comment]
On further inspection, this does not happen for AVS input files, but for LaGriT input files (`-lg`).

**Comment by daniellivingston** [comment]
During write, each line is represented as an 80 char array `elemeTOUGH2`:

```
      elemeTOUGH2 = repeat(' ', 80)
      elemeTOUGH2(1:5) = elne
      elemeTOUGH2(16:20) = ma
```

The variable `ma`, at a higher level, is one row of a character array of length `num_points x 5`. This array was initialized, but not modified, when LaGriT was selected as an input option.

Because the array was 'empty', this manifested itself as garbage characters during TOUGH2 output.

**Comment by daniellivingston** [comment]
Fixed in 34d4a23b09ba29998ebfc09b6bc93ad5cb01629f.
